src/fairseq2/recipes/lm/_online_finetune/_recipe.py

```diff
@@ -284,7 +288,16 @@ def load_online_finetuner(
     # go over actor configs and initialize all of them
     for actor_config in config.vllm.ray_actors:
         log.info(f"Setting up '{actor_config.ray_actor_name}' vllm actor")
-        actor = RemoteVllmModelHandler().create(gangs=gangs, actor_config=actor_config)
+        # check if hf_engine_args is var in actor_config
+        if isinstance(actor_config, HFRayActorConfig):
+            actor = RemoteHFModelHandler().create(
+                gangs=gangs, actor_config=actor_config
+            )
```

Comment:
I don't think this is the best way to go about this. But I'm not sure how else to do it.